docs/user/interactive_basics.rst

```diff
@@ -0,0 +1,175 @@
+.. _interactive_basics:
+
+
+Interactive Environment Basics
+==============================
+
+What is an Interactive Environment?
+-----------------------------------
+
+Interactive environments on RenkuLab are web-based user interfaces (like JupyterLab
+and RStudio) that you can launch to develop and run your code and data workflows.
+They're commonly used for exploratory analysis because you can try out short blocks
+of code before combining everything into a (reproducible) workflow.
+
+You can run your own interactive environments independently from RenkuLab, but
```

Comment:
is that getting too specific @ableuler ?

Reply:
Yeah, maybe... I usually mention it when I present Renku, also because it is a distinguishing feature from other projects which offer a similar workflow. But we can leave it the way it is, that's fine too.

Reply:
hmm that's a good point. Ok I'll include it.